test/fixtures/tasks.yml

```diff
@@ -0,0 +1,13 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  id: 2022-07-18 11:29:09
+  name: MyString
+  description: MyString
+  deadline: 2022-07-18 11:29:09
+
+two:
```

Comment:
Same comments as for `one` fixture, plus could be the deadline date different, so we don't have two fixtures with same values